automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -243,13 +241,16 @@ trait LeonardoTestUtils
       }
 
     val runningOrErroredCluster = Try {
+      implicit val patienceConfig: PatienceConfig =
+        if (stopAfterCreate) clusterStopAfterCreatePatience else clusterPatience
       eventually {
         val cluster = Leonardo.cluster.get(googleProject, clusterName)
         verifyCluster(cluster, googleProject, clusterName, expectedStatuses, clusterRequest, true)
       }
     }
     // Save the cluster init log file whether or not the cluster created successfully
-    saveDataprocLogFiles(creatingCluster).unsafeRunSync()
+    implicit val patienceConfig = downloadDataprocLogsPatience
```

Comment:
shall we name the implicit a bit more unique..just in case conflicts in implicit namings